tests/unit/amo/components/TestUserProfile.js

```diff
@@ -807,35 +802,21 @@ describe(__filename, () => {
     );
     expect(root.find('.UserProfile-reviews')).toHaveProp('footer', null);
 
-    expect(root.find('.AddonReviewListItem')).toHaveLength(reviews.length);
-
-    const firstReview = root.find('.AddonReviewListItem');
+    expect(root.find(UserReview)).toHaveLength(reviews.length);
 
-    expect(firstReview.find('.AddonReviewListItem-body').html()).toContain(
-      fakeReview.body,
-    );
+    const byLine = shallow(root.find(UserReview).prop('byLine'), {
```

Comment:
If you change this to `shallow(<div>{root.find(UserReview).prop('byLine')}</div>)` I think it will work without needing the `store` context.

Reply:
mmh, nope because the `byLine` is the `Link` component, which requires a `store`.